indexer/feature.cpp

```diff
@@ -475,12 +474,59 @@ uint32_t FeatureType::ParseGeometry(int scale)
     }
     m_parsed.m_points = true;
   }
-  return sz;
```

Comment:
Did you find that it was not used anywhere?

Reply:
yeap, it was used in stats code only